### PR TITLE
Create mahmud.json

### PR DESCRIPTION
**Pull request by JBAxB8h7**

### CODE REVIEW

**Comment by Tweak4141** [comment]
JSON file needs to match subdomain name.

File: sub-logs/mahmud.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "repo": "https://jbaxb8h7.github.io/Nazrul.github.io/",
+
+  "owner": {
+    "username": "JBAxB8h7",
+    "email": "kanieloutis758@gmail.com"
+  },
+
+  "target": {
+    "CNAME": {
+      "name": "JBAxB8h7",
```
Comment (WilliamDavidHarrison) [inline sub-logs/mahmud.json:11]: This needs to match the JSON file name.

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

File: sub-logs/mahmud.json (hunk 1 above)
Comment (WilliamDavidHarrison) [reply sub-logs/mahmud.json:11]:
```suggestion
      "name": "mahmud", 
```

**Review by JBAxB8h7 — COMMENTED** [review]
updated cname

**Review by WilliamDavidHarrison — APPROVED** [review]

**Review by Tweak4141 — APPROVED** [review]

**Pull request merged**